### PR TITLE
Add more spiders

### PR DESCRIPTION
**Pull request by qnVictoria**
New spiders: 
- homebase spider
- allo_ua spider

### CODE REVIEW

File: examples/lib/shops/spiders/allo_ua.ex [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    ]
+  end
+
+  @impl Crawly.Spider
+  def parse_item(response) do
+    # Parse response body to document
+    {:ok, document} = Floki.parse_document(response.body)
+
+    # Extract individual product page URLs
+    urls =
```
Comment (oltarasenko) [inline examples/lib/shops/spiders/allo_ua.ex:42]: Could you also extract links from pagination here?

File: examples/lib/shops/spiders/homebase.ex [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      |> Enum.uniq()
+      |> Enum.map(&build_absolute_url/1)
+      |> Enum.map(&Crawly.Utils.request_from_url/1)
+
+    # Create item (for pages where items exists)
+    item = %{
+      title: product_title(document),
+      sku: product_sku(document),
+      price: product_price(document),
+      image: product_image(document)
```
Comment (oltarasenko) [inline examples/lib/shops/spiders/homebase.ex:61]: Can we have a link to image here, instead of the path to the image on file system?

**Review by oltarasenko — COMMENTED** [review]
Looks good. Two minor fixes are requested

File: examples/lib/shops/spiders/homebase.ex (hunk 2 above)
Comment (qnVictoria) [reply examples/lib/shops/spiders/homebase.ex:61]: changed

File: examples/lib/shops/spiders/allo_ua.ex (hunk 1 above)
Comment (qnVictoria) [reply examples/lib/shops/spiders/allo_ua.ex]: added

**Review by oltarasenko — APPROVED** [review]

**Pull request merged**